src/misc/jwt.v1.yaml

```diff
@@ -0,0 +1,5 @@
+$schema: http://json-schema.org/draft-04/schema#
+
+title: Token
+type: string
```

Comment:
link to where the regex comes from?
and is it worth it to lock down?

Reply:
I have found a better regex: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/master/src/System.IdentityModel.Tokens.Jwt/JwtConstants.cs#L58